### PR TITLE
Update project description and add access log documentation

### PR DESCRIPTION
**Pull request by mrproliu**

### CODE REVIEW

File: docs/en/setup/configuration/accesslog.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Access Log Module
```
Comment (wu-sheng) [inline docs/en/setup/configuration/accesslog.md:1]: You miss the menu item for this new doc

**Comment by wu-sheng** [comment]
https://skywalking.apache.org/docs/skywalking-rover/next/en/setup/overview/#skywalking-oap This part should be updated as well.

**Comment by wu-sheng** [comment]
And Configuration and Use Cases should be updated too.

Meanwhile, I can only see deploy on Linux, where is on K8s? The access log requires k8s, right?

**Comment by mrproliu** [comment]
Yes, I will add a new Kubernetes deploy documentation.

**Comment by wu-sheng** [comment]
I think from now on, running on k8s is the first and most important use case, until we are going to monitor VM service through eBPF(but not now).

The previous use cases seem to be a little advanced features only.

File: docs/en/setup/overview.md [hunk 2]
```diff
@@ -15,7 +15,8 @@ The SkyWalking Rover requires specialized protocols to communicate with SkyWalki
 
 | SkyWalking Rover Version | SkyWalking OAP |
 |--------------------------|----------------|
-|0.1.0+                    | \> = 9.1.0     |
+| 0.1.0+                   | \> = 9.1.0     |
```
Comment (wu-sheng) [inline docs/en/setup/overview.md:18]: Let's remove 0.1, after all, we changed the landscape of the project in 0.6. We should make this changing very clear in the release note.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**